src/components/modal_window/modalWindow.js

```diff
@@ -0,0 +1,22 @@
+export const modalWindow = () => {
+    
+    const overlay = document.createElement("div");
+    overlay.classList.add("overlay");
+    document.getElementById("app").appendChild(overlay);
+    
+    const modal = document.createElement("div");
+    modal.classList.add("modal-window");
+    document.getElementById("app").appendChild(modal);
+    
+    const btn = document.createElement("button");
+    btn.classList.add("btn");
+    btn.innerHTML = "&times;"
+    modal.appendChild(btn);
+    
+    const closeModal = () => {
+        modal.style.display = "none";
+        overlay.style.display = "none"
+    }
+    
+    btn.addEventListener("click", closeModal);
+}
```

Comment:
The component must accept a callback function. The call is supposed to display the value of the callback inside the modal.